.env.sample

```
@@ -0,0 +1,11 @@
+DATABASE_URL : "mysql://username:userpassword@127.0.0.1:3306/database_name"
```

```suggestion
DATABASE_URL : "rdbms://username:userpassword@localhost:db_port/database_name"
```
다른 값들도 중의적으로 표현되면 좋을 것 같습니다-!